check-plugins/check_redis_sentinel_cluster/check_redis_sentinel_cluster.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Script for checking the health of a Redis Sentinel cluster.
+
+Copyright 2024 Deutsche Telekom MMS GmbH
+Maintainer: Daniel Uhlmann & Andreas Hering
+
+This script is used to monitor and ensure the health of a Redis Sentinel cluster.
+It checks the availability of the master, the replicas, and verifies that the
+number of healthy replicas matches the expected count.
+
+Usage:
+    python3 check_redis_sentinel.py -u <sentinel_url> -p <sentinel_port>
+              -t <sentinel_auth_token> -r <healthy_replica_count> -n <redis_primary_name>
+
+Arguments:
```

Comment:
Do we need to have the arguments here and in the check_redis_sentinel-function when argparse automatically displays them when running the script with `--help`?

I'd say let's remove them here and in the function and only keep them in the README.

Reply:
I have removed the code you mentioned.